swagger.json

```diff
@@ -3194,17 +3194,29 @@
                     "description": "Group of the driver.",
                     "example": 111
                   },
-                  "activeHours": {
+                  "distanceMiles": {
                     "type": "number",
                     "format": "float",
-                    "description": "Hours spent on duty or driving, rounded to two decimal places.",
+                    "description": "Distance driven in miles, rounded to two decimal places.",
+                    "example": 123.24
+                  },
+                  "activeMs": {
+                    "type": "number",
+                    "format": "float",
+                    "description": "Milliseconds spent on duty or driving",
                     "example": 5.40
                   },
-                  "distanceMiles": {
+                  "onDutyMs": {
                     "type": "number",
                     "format": "float",
-                    "description": "Distance driven in miles, rounded to two decimal places.",
-                    "example": 123.24
+                    "description": "Milliseconds spent on duty",
```

Comment:
Nit: `Duration in milliseconds then driver was ON DUTY during the requested time range.`